src/app/my-page/profile/panel/ProfileBioEditor.tsx

```diff
@@ -50,6 +56,8 @@ const ProfileBioEditor = ({
     data.profileImageUrl,
   )
   const [imageChanged, setImageChanged] = useState<boolean>(false)
+  const imageRef = useRef<HTMLImageElement>(null)
+  const [cropper, setCropper] = useState<any>(null)
```

Comment:
이부분 이렇게 타입 지정할 수 있을 것 같아용
```suggestion
  const [cropper, setCropper] = useState<Cropper | null>(null)
```